### ISSUE TITLE
训练实例分割时总是会自动退出

### ISSUE THREAD

**Issue by yipsilon** [PaddlePaddle/PaddleDetection#9410]
### 问题确认 Search before asking

- [x] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleDetection/issues)，没有发现相似的bug。I have searched the [issues](https://github.com/PaddlePaddle/PaddleDetection/issues) and found no similar bug report.

### Bug组件 Bug Component

Training

### Bug描述 Describe the Bug

使用官方的样本数据集 instance_seg_coco_examples 做的训练测试，经过跟踪问题出在 ppdet/engine/trainer.py 中的 outputs = model(data) 这里，感觉像是加载模型时出现的异常，在 windows 事件查看器里有如下提示：

> 出错应用程序名称： python.exe，版本： 3.12.11150.1013，时间戳： 0x684194d7
> 出错模块名称： libpaddle.pyd， 版本： 0.0.0.0，时间戳： 0x685b8f57
> 异常代码： 0xc0000005
> 错误偏移： 0x00000000066a0f0e
> 出错进程 ID： 0x3C6C
> 出错应用程序开始时间： 0x1DC0139AA60AF48
> Faulting 应用程序路径： E:\Anaconda3\envs\PaddleDetection-281\python.exe
> Faulting 模块路径： E:\Anaconda3\envs\PaddleDetection-281\Lib\site-packages\paddle\base\libpaddle.pyd
> Report ID： 95f8bb8c-9d72-4726-ba21-681c3d697bf0
> Faulting 包全名： 
> Faulting 程序包相对应用程序 ID：  个

PaddlePaddle 版本为 CPU 3.1.0，PaddleDetection 为 2.8.1。

模型配置文件为：

> epoch: 50
> debug: true
> use_gpu: false
> use_xpu: false
> use_mlu: false
> use_npu: false
> log_iter: 10
> save_dir: output
> snapshot_epoch: 10
> target_metrics: mask
> print_flops: false
> print_params: false
> use_ema: True
> ema_decay: 0.9999
> ema_decay_type: "exponential"
> ema_filter_no_grad: True
> 
> metric: COCO
> num_classes: 100
> worker_num: 2
> 
> TrainDataset:
>   name: COCOInstSegDataset
>   image_dir: images
>   anno_path: annotations/instance_train.json
>   dataset_dir: ./workspace/labelme/COCOInstSegDataset
>   data_fields: ['image', 'gt_bbox', 'gt_class', 'gt_poly', 'is_crowd']
> 
> EvalDataset:
>   name: COCOInstSegDataset
>   image_dir: images
>   anno_path: annotations/instance_val.json
>   dataset_dir: ./workspace/labelme/COCOInstSegDataset
> 
> TestDataset:
>   name: ImageFolder
>   anno_path: annotations/instance_val.json
>   dataset_dir: ./workspace/labelme/COCOInstSegDataset
> 
> TrainReader:
>   sample_transforms:
>     - Decode: {}
>     - Poly2Mask: {del_poly: True}
>     - RandomDistort: {prob: 0.8}
>     - RandomExpand: {fill_value: [123.675, 116.28, 103.53]}
>     - RandomCrop: {prob: 0.8}
>     - RandomFlip: {}
>   batch_transforms:
>     - BatchRandomResize: {target_size: [480, 512, 544, 576, 608, 640, 640, 640, 672, 704, 736, 768, 800], random_size: True, random_interp: True, keep_ratio: False}
>     - NormalizeImage: {mean: [0., 0., 0.], std: [1., 1., 1.], norm_type: none}
>     - NormalizeBox: {}
>     - BboxXYXY2XYWH: {}
>     - Permute: {}
>   batch_size: 1
>   shuffle: true
>   drop_last: true
>   collate_batch: false
>   use_shared_memory: false
>   use_process: false
> 
> EvalReader:
>   sample_transforms:
>     - Decode: {}
>     - Resize: {target_size: [640, 640], keep_ratio: False, interp: 2}
>     - NormalizeImage: {mean: [0., 0., 0.], std: [1., 1., 1.], norm_type: none}
>     - Permute: {}
>   batch_size: 1 # mask be 1
>   shuffle: false
>   drop_last: false
> 
> TestReader:
>   inputs_def:
>     image_shape: [3, 640, 640]
>   sample_transforms:
>     - Decode: {}
>     - Resize: {target_size: [640, 640], keep_ratio: False, interp: 2}
>     - NormalizeImage: {mean: [0., 0., 0.], std: [1., 1., 1.], norm_type: none}
>     - Permute: {}
>   batch_size: 1
>   shuffle: false
>   drop_last: false
> 
> LearningRate:
>   base_lr: 0.0001
>   schedulers:
>   - !PiecewiseDecay
>     gamma: 1.0
>     milestones: [100]
>     use_warmup: true
>   - !LinearWarmup
>     start_factor: 0.001
>     steps: 2000
> 
> OptimizerBuilder:
>   clip_grad_by_norm: 0.1
>   regularizer: false
>   optimizer:
>     type: AdamW
>     weight_decay: 0.0001
> 
> architecture: DETR
> with_mask: True
> pretrain_weights: "./workspace/labelme/Mask-RT-DETR-H_pretrained.pdparams"
> norm_type: sync_bn
> hidden_dim: 256
> use_focal_loss: True
> eval_size: [640, 640]
> num_prototypes: 128
> find_unused_parameters: True
> 
> DETR:
>   backbone: PPHGNetV2
>   neck: MaskHybridEncoder
>   transformer: MaskRTDETR
>   detr_head: MaskDINOHead
>   post_process: DETRPostProcess
> 
> PPHGNetV2:
>   arch: 'H'
>   return_idx: [0, 1, 2, 3]
>   freeze_stem_only: True
>   freeze_at: 0
>   freeze_norm: True
>   lr_mult_list: [0., 0.01, 0.01, 0.01, 0.01]
> 
> MaskHybridEncoder:
>   hidden_dim: 512
>   use_encoder_idx: [3]
>   num_encoder_layers: 2
>   encoder_layer:
>     name: TransformerLayer
>     d_model: 512
>     nhead: 8
>     dim_feedforward: 2048
>     dropout: 0.
>     activation: 'gelu'
>   expansion: 1.0
>   mask_feat_channels: [256, 128]
> 
> MaskRTDETR:
>   num_queries: 300
>   position_embed_type: sine
>   feat_strides: [8, 16, 32]
>   num_levels: 3
>   nhead: 8
>   dim_feedforward: 1024
>   dropout: 0.0
>   activation: relu
>   num_denoising: 100
>   label_noise_ratio: 0.5
>   box_noise_scale: 1.0
>   learnt_init_query: False
>   mask_enhanced: True
> 
> MaskDINOHead:
>   loss:
>     name: MaskDINOLoss
>     loss_coeff: {class: 4, bbox: 5, giou: 2, mask: 5, dice: 5}
>     aux_loss: True
>     use_vfl: True
>     vfl_iou_type: 'mask'
>     matcher:
>       name: HungarianMatcher
>       matcher_coeff: {class: 4, bbox: 5, giou: 2, mask: 5, dice: 5}
> 
> DETRPostProcess:
>   num_top_queries: 100
>   mask_stride: 4
> 
> export:
>   post_process: True
>   nms: True
>   benchmark: False
>   fuse_conv_bn: False

下面是 Windows 生成的错误报告，感觉两个错误的原因应该是相同的。
[Report.txt](https://github.com/user-attachments/files/21522037/Report.txt)

### 复现环境 Environment

- OS: Windows 11
- PaddlePaddle: 3.1.0 CPU
- PaddleDetection: release/2.8.1
- Python: 3.12.11
- CPU: AMD 5590X
- RAM: 128G

### Bug描述确认 Bug description confirmation

- [x] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [ ] 我愿意提交PR！I'd like to help by submitting a PR!

**Comment by BluebirdStory** [comment]
可以拉取最新的分支试一下。

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
